rich/traceback.py

```diff
@@ -243,8 +243,11 @@ def __init__(
         self.locals_max_length = locals_max_length
         self.locals_max_string = locals_max_string
 
-        self.suppress: Sequence[str] = []
+        self.suppress: Sequence[str, callable] = []
```

Comment:
Suspect his should be `Union[str, Callable]` (note Callable imported from the typing module).

Reply:
ok. will do.  FYI, I got that lower case spelling from a doc entry in the same file under from_exception.